tests/src/Rules/ThrowsPhpDocRuleTest.php

```diff
@@ -69,6 +69,7 @@ protected function getRule(): Rule
 			$this->createThrowsAnnotationReader(),
 			$this->createBroker(),
 			$this->reportUnusedCatchesOfUncheckedExceptions,
+			true,
```

Comment:
This configuration is test-specific, should we use a flag property here?

Reply:
You're right. I've changed it